### PR TITLE
Bluetooth: Increase CONFIG_BT_HCI_TX_STACK_SIZE for BT_SPI

### PR DESCRIPTION
**Pull request by pfalcon**
Older value of 256 was introduced in 1.7 times. Testing on
BOARD=96b_carbon with 2 weeks old codebase however showed that TX
thread stack however can grow to 324 bytes. Finally, with the
latest master, following stacks are reported on BLE disconnect
(with CONFIG_INIT_STACKS enabled):

rx stack (real size 1024): unused 452	usage 572 / 1024 (55 %)
tx stack (real size 384):  unused 16	usage 368 / 384 (95 %)

Two outcomes:

1. TX stack needs increase.
2. Over time, the stack usage grows, plus variations in SPI
drivers should also be taken into account.

So, increase the stack size to 416 bytes, to leave some headroom
beyond the immediate values seen on 96b_carbon.

Jira: ZEP-2510

Signed-off-by: Paul Sokolovsky <paul.sokolovsky@linaro.org>

### CODE REVIEW

**Review by jhedberg — DISMISSED** [review]

**Comment by pfalcon** [comment]
A follow up of yesterday's IRC discussion, later captured as https://jira.zephyrproject.org/browse/ZEP-2510 . @mbolivar (as a maintainer of 96b_carbon) OKed this change in the Jira, bit mentioned he might be on vacation next few days, so hopefully this can be merged sooner than that.

**Review by carlescufi — DISMISSED** [review]

File: subsys/bluetooth/host/Kconfig
Comment (Vudentz) [inline subsys/bluetooth/host/Kconfig]: Perhaps we could go with 512 then, it should give a little more space for the driver to grow.

Comment (pfalcon) [inline subsys/bluetooth/host/Kconfig]: Well, I see that BT stacks are finely managed (e.g. 256 bytes here, 446 bytes for BT_CTLR_RX_PRIO_STACK_SIZE), that I followed that approach, just giving explanation why I didn't set it to "almost rount" 384 bytes. If it's really grounded, we can set it to 512. One concern is that BT_HCI_TX_STACK_SIZE isn't user-configurable, so not easy to optimize for a particular config (I actually plan to submit a ticket on that).

**Comment by jhedberg** [comment]
It seems CI failed since the commit message contains tab characters (coming from a copy-paste of console output). I think it's a CI bug (it shouldn't be that strict), but it's your choice do you want to fix that in the commit message or wait for CI to be fixed (or for someone like @nashif to override and merge this anyway)

**Comment by pfalcon** [comment]
@jhedberg : Fixed tabs in the commit message, the build now passes. Thanks.

**Review by jhedberg — APPROVED** [review]

**Pull request merged**
